src/main/java/uk/gov/hmcts/reform/pip/data/management/errorhandling/GlobalExceptionHandler.java

```diff
@@ -77,6 +78,19 @@
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
     }
 
+
+    @ExceptionHandler(DateHeaderValidationException.class)
+    public ResponseEntity<ExceptionResponse> handle(DateHeaderValidationException ex) {
```

Comment:
were almost stepping on the toes of the above handler, so maybe a `HeaderValidationException` to be handled and then the exceptions can extend that

Reply:
fixed